.circleci/config.yml

```diff
@@ -167,6 +167,7 @@ jobs:
   loadtest:
     executor: golang
     parallelism: 10
+    resource_class: medium+
```

Comment:
Is there any downside to this?

Reply:
I don't think so. I've seen bigger machines used in other open-telemetry repos

Reply:
Should we also reduce the number of shards if we have larger machines?

Reply:
Do you mean number of tests runners? That makes sense, reduced it from 10 to 6 (machine size increase is +50%)

Reply:
Actually, it's 50% slower now because e2e tests have specific time to run (15s each). So I'd better to bring number of containers back to 10

Reply:
@bogdandrutu @tigrannajaryan what do you think ^ ?

Reply:
Fine either way. We are going to have more tests over time, there is no need to optimize for any particular number of tests.

Reply:
I'll bring it back to 10 then